src/pudl/package_data/vcerare/__init__.py

```diff
@@ -0,0 +1 @@
+"""CSV file extraction maps for VCE RARE generation profiles."""
```

Comment:
Just a note for consistency if you want to change this to say "Power Dataset" to match the others.